backend/routes/recipeRouter.js

```diff
@@ -30,7 +30,11 @@ router.get( '/:id', (req, res) => {
 
 
 /* POST */
+router.post( '/', (req, res) => {
+  const recipeData = req.body;
 
+  recipes.insert(recipeData).then( (newId) => {res.json(newId);}).catch( (err) => {res.status(500).json({ error: err})});
```

Comment:
Make sure to organize this line consistently. If you have to block it out then do that 👍 looks better

Reply:
Oh definitely! This was just a quick line to use while working out the helpers!